docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -425,6 +425,17 @@ arguments.
         by specifying a slide-window size ``s``, where ``s`` can an integer or float. GSProcessing then transforms each
         numeric value ``v`` of the property into a range from ``v - s/2`` through ``v + s/2`` , and assigns the value v
         to every bucket that the range covers.
+
+-  ``categorical``
+
+   -  Transforms one value from a fixed list of possible values which can be treated as a categorical feature which are encoded using one-hot encoding. No kwargs required for categorical feature.
+
+-  ``multi-categorical``
+
+   -  Apply column-wise transformation for vector-like data from a fixed list of possible values which can be treated as a categorical feature which are encoded using multi-hot encoding.
+   -  ``kwargs``:
+      - ``separator`` (String, optional): Same as the one in the No-op operation, the separator is used to
+        split input value in CSV format. If it is not provided, then the whole value will be considered as an array.
 --------------
```

Comment:
Let's create a note about hard-coded limit of 100 categories, as in https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-feature-encoding.html#machine-learning-categorical-features

Use the `.. note::` rst env: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html